2021-05-26 00:22:45,059 INFO    MainThread:18204 [internal.py:wandb_internal():91] W&B internal server running at pid: 18204, started at: 2021-05-26 00:22:45.054351
2021-05-26 00:22:45,061 INFO    WriterThread:18204 [datastore.py:open_for_write():77] open: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210526_002244-23oo874y\run-23oo874y.wandb
2021-05-26 00:22:45,062 DEBUG   HandlerThread:18204 [handler.py:handle_request():121] handle_request: check_version
2021-05-26 00:22:45,062 DEBUG   SenderThread:18204 [sender.py:send():176] send: header
2021-05-26 00:22:45,063 DEBUG   SenderThread:18204 [sender.py:send_request():186] send_request: check_version
2021-05-26 00:22:45,179 DEBUG   SenderThread:18204 [sender.py:send():176] send: run
2021-05-26 00:22:45,199 DEBUG   SenderThread:18204 [git.py:repo():33] git repository is invalid
2021-05-26 00:22:45,435 DEBUG   HandlerThread:18204 [handler.py:handle_request():121] handle_request: run_start
2021-05-26 00:22:45,437 INFO    SenderThread:18204 [dir_watcher.py:__init__():168] watching files in: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210526_002244-23oo874y\files
2021-05-26 00:22:45,437 INFO    SenderThread:18204 [sender.py:_start_run_threads():699] run started: 23oo874y with start time 1621981364
2021-05-26 00:22:45,437 DEBUG   SenderThread:18204 [sender.py:send():176] send: summary
2021-05-26 00:22:45,438 INFO    SenderThread:18204 [sender.py:_save_file():821] saving file wandb-summary.json with policy end
2021-05-26 00:22:46,439 INFO    Thread-12 :18204 [dir_watcher.py:_on_file_created():216] file/dir created: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210526_002244-23oo874y\files\wandb-summary.json
2021-05-26 00:22:47,176 DEBUG   HandlerThread:18204 [meta.py:__init__():39] meta init
2021-05-26 00:22:47,176 DEBUG   HandlerThread:18204 [meta.py:__init__():53] meta init done
2021-05-26 00:22:47,176 DEBUG   HandlerThread:18204 [meta.py:probe():210] probe
2021-05-26 00:22:47,184 DEBUG   HandlerThread:18204 [git.py:repo():33] git repository is invalid
2021-05-26 00:22:47,184 DEBUG   HandlerThread:18204 [meta.py:_save_pip():57] save pip
2021-05-26 00:22:47,185 DEBUG   HandlerThread:18204 [meta.py:_save_pip():71] save pip done
2021-05-26 00:22:47,185 DEBUG   HandlerThread:18204 [meta.py:_save_conda():78] save conda
2021-05-26 00:22:47,439 INFO    Thread-12 :18204 [dir_watcher.py:_on_file_created():216] file/dir created: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210526_002244-23oo874y\files\requirements.txt
2021-05-26 00:22:47,440 INFO    Thread-12 :18204 [dir_watcher.py:_on_file_created():216] file/dir created: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210526_002244-23oo874y\files\conda-environment.yaml
2021-05-26 00:22:50,288 DEBUG   HandlerThread:18204 [meta.py:_save_conda():86] save conda done
2021-05-26 00:22:50,288 DEBUG   HandlerThread:18204 [meta.py:probe():252] probe done
2021-05-26 00:22:50,291 DEBUG   SenderThread:18204 [sender.py:send():176] send: files
2021-05-26 00:22:50,291 INFO    SenderThread:18204 [sender.py:_save_file():821] saving file wandb-metadata.json with policy now
2021-05-26 00:22:50,300 DEBUG   HandlerThread:18204 [handler.py:handle_request():121] handle_request: stop_status
2021-05-26 00:22:50,300 DEBUG   SenderThread:18204 [sender.py:send_request():186] send_request: stop_status
2021-05-26 00:22:50,441 INFO    Thread-12 :18204 [dir_watcher.py:_on_file_modified():229] file/dir modified: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210526_002244-23oo874y\files\conda-environment.yaml
2021-05-26 00:22:50,441 INFO    Thread-12 :18204 [dir_watcher.py:_on_file_created():216] file/dir created: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210526_002244-23oo874y\files\output.log
2021-05-26 00:22:50,441 INFO    Thread-12 :18204 [dir_watcher.py:_on_file_created():216] file/dir created: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210526_002244-23oo874y\files\wandb-metadata.json
2021-05-26 00:22:50,899 INFO    Thread-15 :18204 [upload_job.py:push():133] Uploaded file C:\Users\ROBERT~1\AppData\Local\Temp\tmpgs6j11xmwandb\13qtd431-wandb-metadata.json
2021-05-26 00:23:05,492 DEBUG   HandlerThread:18204 [handler.py:handle_request():121] handle_request: stop_status
2021-05-26 00:23:05,492 DEBUG   SenderThread:18204 [sender.py:send_request():186] send_request: stop_status
2021-05-26 00:23:12,569 DEBUG   HandlerThread:18204 [handler.py:handle_request():121] handle_request: pause
2021-05-26 00:23:12,569 INFO    HandlerThread:18204 [handler.py:handle_request_pause():519] stopping system metrics thread
2021-05-26 00:23:12,609 DEBUG   SenderThread:18204 [sender.py:send():176] send: stats
2021-05-26 00:23:20,700 DEBUG   HandlerThread:18204 [handler.py:handle_request():121] handle_request: stop_status
2021-05-26 00:23:20,700 DEBUG   SenderThread:18204 [sender.py:send_request():186] send_request: stop_status
2021-05-26 00:23:36,129 DEBUG   HandlerThread:18204 [handler.py:handle_request():121] handle_request: stop_status
2021-05-26 00:23:36,129 DEBUG   SenderThread:18204 [sender.py:send_request():186] send_request: stop_status
2021-05-26 00:23:51,312 DEBUG   HandlerThread:18204 [handler.py:handle_request():121] handle_request: stop_status
2021-05-26 00:23:51,312 DEBUG   SenderThread:18204 [sender.py:send_request():186] send_request: stop_status
2021-05-26 00:24:06,510 DEBUG   HandlerThread:18204 [handler.py:handle_request():121] handle_request: stop_status
2021-05-26 00:24:06,510 DEBUG   SenderThread:18204 [sender.py:send_request():186] send_request: stop_status
2021-05-26 00:24:21,709 DEBUG   HandlerThread:18204 [handler.py:handle_request():121] handle_request: stop_status
2021-05-26 00:24:21,709 DEBUG   SenderThread:18204 [sender.py:send_request():186] send_request: stop_status
2021-05-26 00:24:37,507 DEBUG   HandlerThread:18204 [handler.py:handle_request():121] handle_request: stop_status
2021-05-26 00:24:37,507 DEBUG   SenderThread:18204 [sender.py:send_request():186] send_request: stop_status
2021-05-26 00:24:52,689 DEBUG   HandlerThread:18204 [handler.py:handle_request():121] handle_request: stop_status
2021-05-26 00:24:52,689 DEBUG   SenderThread:18204 [sender.py:send_request():186] send_request: stop_status
2021-05-26 00:25:07,889 DEBUG   HandlerThread:18204 [handler.py:handle_request():121] handle_request: stop_status
2021-05-26 00:25:07,889 DEBUG   SenderThread:18204 [sender.py:send_request():186] send_request: stop_status
2021-05-26 00:25:23,087 DEBUG   HandlerThread:18204 [handler.py:handle_request():121] handle_request: stop_status
2021-05-26 00:25:23,087 DEBUG   SenderThread:18204 [sender.py:send_request():186] send_request: stop_status
2021-05-26 00:25:38,303 DEBUG   HandlerThread:18204 [handler.py:handle_request():121] handle_request: stop_status
2021-05-26 00:25:38,303 DEBUG   SenderThread:18204 [sender.py:send_request():186] send_request: stop_status
2021-05-26 00:25:42,158 WARNING MainThread:18204 [internal.py:is_dead():367] Internal process exiting, parent pid 1984 disappeared
2021-05-26 00:25:42,158 ERROR   MainThread:18204 [internal.py:wandb_internal():143] Internal process shutdown.
2021-05-26 00:25:42,308 INFO    HandlerThread:18204 [handler.py:finish():632] shutting down handler
2021-05-26 00:25:42,489 INFO    SenderThread:18204 [sender.py:finish():925] shutting down sender
2021-05-26 00:25:42,489 INFO    SenderThread:18204 [dir_watcher.py:finish():282] shutting down directory watcher
2021-05-26 00:25:42,529 INFO    SenderThread:18204 [dir_watcher.py:finish():312] scan: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210526_002244-23oo874y\files
2021-05-26 00:25:42,529 INFO    SenderThread:18204 [dir_watcher.py:finish():318] scan save: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210526_002244-23oo874y\files\conda-environment.yaml conda-environment.yaml
2021-05-26 00:25:42,530 INFO    SenderThread:18204 [dir_watcher.py:finish():318] scan save: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210526_002244-23oo874y\files\config.yaml config.yaml
2021-05-26 00:25:42,530 INFO    SenderThread:18204 [dir_watcher.py:finish():318] scan save: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210526_002244-23oo874y\files\output.log output.log
2021-05-26 00:25:42,536 INFO    SenderThread:18204 [dir_watcher.py:finish():318] scan save: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210526_002244-23oo874y\files\requirements.txt requirements.txt
2021-05-26 00:25:42,537 INFO    SenderThread:18204 [dir_watcher.py:finish():318] scan save: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210526_002244-23oo874y\files\wandb-metadata.json wandb-metadata.json
2021-05-26 00:25:42,537 INFO    SenderThread:18204 [dir_watcher.py:finish():318] scan save: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210526_002244-23oo874y\files\wandb-summary.json wandb-summary.json
2021-05-26 00:25:42,538 INFO    SenderThread:18204 [file_pusher.py:finish():176] shutting down file pusher
2021-05-26 00:25:42,538 INFO    SenderThread:18204 [file_pusher.py:join():181] waiting for file pusher
2021-05-26 00:25:42,680 INFO    WriterThread:18204 [datastore.py:close():258] close: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210526_002244-23oo874y\run-23oo874y.wandb
2021-05-26 00:25:43,080 INFO    Thread-17 :18204 [upload_job.py:push():133] Uploaded file C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210526_002244-23oo874y\files\config.yaml
2021-05-26 00:25:43,122 INFO    Thread-18 :18204 [upload_job.py:push():133] Uploaded file C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210526_002244-23oo874y\files\requirements.txt
2021-05-26 00:25:43,293 INFO    Thread-16 :18204 [upload_job.py:push():133] Uploaded file C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210526_002244-23oo874y\files\conda-environment.yaml
2021-05-26 00:25:43,549 INFO    Thread-19 :18204 [upload_job.py:push():133] Uploaded file C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210526_002244-23oo874y\files\wandb-summary.json
2021-05-26 00:25:44,199 INFO    MainThread:18204 [internal.py:handle_exit():78] Internal process exited
